examples/hybrid/driver.jl

```diff
@@ -127,19 +127,9 @@ if config.parsed_args["check_conservation"]
     @info "    Net mass change / total mass: $mass_conservation"
     @info "    Net water change / total water: $water_conservation"
 
-    sfc = p.atmos.surface_model
-
-    if CA.has_no_source_or_sink(config.parsed_args)
-        @test energy_conservation ≈ 0 atol = 50 * eps(FT)
-        @test mass_conservation ≈ 0 atol = 50 * eps(FT)
-        @test water_conservation ≈ 0 atol = 50 * eps(FT)
-    else
-        @test energy_conservation ≈ 0 atol = sqrt(eps(FT))
-        @test mass_conservation ≈ 0 atol = sqrt(eps(FT))
-        if sfc isa CA.PrognosticSurfaceTemperature
-            @test water_conservation ≈ 0 atol = sqrt(eps(FT))
-        end
-    end
+    @test energy_conservation ≈ 0 atol = 50 * eps(FT)
+    @test mass_conservation ≈ 0 atol = 50 * eps(FT)
+    @test water_conservation ≈ 0 atol = 50 * eps(FT)
```

Comment:
Removing the case dependent tolerances here seems reasonable - the `Float64` energy conservation result in buildkite is worse than the moisture / mass tests, is it clear why that is?

Reply:
The incorrect branching is due to the boolean and false string in vert_diff. I removed the `has_no_source_or_sink` function as it is not used anywhere now.